contract/models/contract.py

```diff
@@ -237,8 +289,7 @@ def _get_lines_to_invoice(self, date_ref=False):
             lambda l: not l.is_canceled and l.recurring_next_date <= date_ref)
 
     @api.multi
-    def recurring_create_invoice(self):
-        invoice_model = self.env['account.invoice']
+    def _prepare_recurring_invoices_values(self):
```

Comment:
Add a docstring explaining this prepare method as the side effect of updating recurring_next_date

Reply:
done